external/radiation/python/getdata.py

```diff
@@ -0,0 +1,240 @@
+import numpy as np
+from config import *
+import os 
+import xarray as xr 
+import warnings
+
+
+def random_numbers(LOOKUP_DIR, me):
```

Comment:
We typically reserve upper case for module level constants. Can you modify all the function arguments in this file to lower case?

Also, `me` is not an informative name. I would change to `tile` and add type hints to clarify what it is.
```suggestion
def random_numbers(lookup_dir: str, tile: int):
```

You should be able to make similar edits in the other functions.

Reply:
addressed